lib/src/internal/utils.dart

```diff
@@ -8,7 +8,7 @@ import 'package:nyxx_interactions/src/models/interaction.dart';
 import 'package:nyxx_interactions/src/models/slash_command.dart';
 
 /// Slash command names and subcommands names have to match this regex
-final RegExp slashCommandNameRegex = RegExp(r"^[\w-]{1,32}$");
+final RegExp slashCommandNameRegex = RegExp(r"^[-_\p{L}\p{N}\p{sc=Deva}\p{sc=Thai}]{1,32}$", unicode: true);
```

Comment:
why such change?

Reply:
Discord has updated the regex. _commands updated it a while ago, but _interactions is still stuck on the old one...